pages/linux/wodim.md

```diff
@@ -0,0 +1,17 @@
+# wodim
+
+> Command for record/burning data using a optical drive to CD or DVD.
+> Caution when using this command.
```

Comment:
Please have a look at https://github.com/tldr-pages/tldr/pull/1399#issuecomment-307578124 for our discussion on the language and tone for potentially destructive commands.

You can re-word the description along those lines.